src/components/dashboard/ClaimButton.js

```diff
@@ -152,22 +152,22 @@ const ClaimAnimationButton = memo(({ styles, entitlement, nextClaim, onPress, is
 
   const _onPress = useOnPress(onPress)
 
-  if (initialEntitlementRef.current) {
-    return (
-      <ClaimButton
-        {...buttonProps}
-        styles={styles}
-        entitlement={entitlement}
-        nextClaim={nextClaim}
-        onPress={_onPress}
-        isInQueue={isInQueue}
-      />
-    )
-  }
+  // if (initialEntitlementRef.current) {
+  //   return (
+  //     <ClaimButton
+  //       {...buttonProps}
+  //       styles={styles}
+  //       entitlement={entitlement}
+  //       nextClaim={nextClaim}
+  //       onPress={onPress}
```

Comment:
any reason to remove the event wrapped in useOnPress?

Reply:
this is commented deprecated code. in the functional code it still uses _onPress

Reply:
in latest commit iv'e removed it again, because didnt you just remove it in some other PR? saying it crashes?